File: 2.Warmup/2.8. Count Digits/count-digits..js
//Digit Sum
console.log("------ Count Digit ---------");
function countDigit(n){
    if(n === 0) { return 1}
    n = Math.abs(n)
    let counter = 0
    while(n > 0){
        n = Math.floor(n/10)
        counter++
    }
    return counter
}

console.log(countDigit(789))

File: 2.Warmup/2.8. Count Digits/count_digits.py
# Digit Count
print("------ Count Digit ---------")

def digit_count(num):
    if num == 0 :
        return 1
    sign = -1 if num <0 else 1
    num = sign * num
    counter = 0
    while(num > 0):
        num = int(num/10)
        counter += 1
    return counter

print(digit_count(789))